test/IDE/complete_unapplied_static_ref_to_func_with_error.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-ide-test -batch-code-completion -source-filename %s -filecheck %raw-FileCheck -completion-output-dir %t
+
+struct Test {
+  func myMethod() -> Invalid {}
+
+  func test() {
+    Test.myMethod#^COMPLETE^#
```

Comment:
Could you please add a test for global and local functions as well?

Reply:
Done